src/index.ts

```diff
@@ -124,7 +202,7 @@ export const getNPMMetadataForDep = async (dep, npmAuthToken?: string) => {
     } else {
       tableDeets.push({
         name: "License",
-        message: "<b>NO LICENSE FOUND</b>",
+        message: "**NO LICENSE FOUND**",
```

Comment:
I wanted only one place to handle different types of markdown and `<b>` isn't correct on BitBucket. Wouldn't `**` for bold be the most likely to be supported?

Reply:
👍 fine by me